utils/hosted_clusters_chaos_monkey.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+trap '_delete_clusters ${EXECUTION_SEED}' SIGINT SIGTERM
+
+### EDIT VARIABLES
+LOG_FILE="/tmp/chaos_monkey.log"
+NUMBER_OF_CLUSTERS=3
+CLUSTER_NAME_SEED="chaos"
+AWS_SUBNETS="subnet-xxxxxxx,subnet-xxxxxx"
+PROVISION_SHARD="provision_shard_id:xxxxxx"
+export AWS_REGION="us-east-2"
+###
+
+exec 3>&1 1>>${LOG_FILE} 2>&1
+
+log() {
+  message=$1
+  timestamp=$(date +%Y-%m-%d\ %H:%M:%S)
+  echo -e "$timestamp\t$message"
+}
+
+_delete_clusters() {
+  log "INFO: Captured Control-C key, deleting all clusters with ${1} seed" | tee /dev/fd/3
+  for CLUSTER in $(rosa list clusters | grep $1 | awk '{print $1}'); do
+    log "INFO: Deleting cluster ${CLUSTER}" | tee /dev/fd/3
+    _delete_cluster "${CLUSTER}"
+    log "INFO: Cluster ${CLUSTER} deleted" | tee /dev/fd/3
+  done
+  exit 0
+}
+
+_random_time(){
```

Comment:
Can we move away from a random time to a static one? It will be easier to see patterns in a set interval